source/src/ports/generic_networkhandler.h

```diff
@@ -57,12 +57,6 @@ typedef struct {
 
 NetworkStatus g_network_status; /**< Global variable holding the current network status */
 
-/** @brief The platform independent part of network handler initialization routine
```

Comment:
This would be the "correct" function declaration as the doxygen comment is the correct one. The other one at line 65-69 is the "wrong" one

Reply:
Sorry, gave the wrong lines numer 71-75 is the one with the wrong description